Help Desk/helpDesk_editTechnicianGroupProcess.php

```diff
@@ -45,6 +45,9 @@
 
         $groupName = $_POST['groupName'] ?? '';
         $departmentID = $_POST['departmentID'] ?? null;
+        if (empty($departmentID)) {
```

Comment:
This seems redundant, given the null coalesce above...unless I've missed something...

Reply:
It does seem like its redundant. Its minor and shouldn't affect anything, so we'll package it in the next release. Thanks for the heads up!

Reply:
Sounds good,t hanks : )